.github/workflows/test.yaml

```diff
@@ -1,17 +1,16 @@
-name: Push
-on: [push]
+# Test flow triggered by other workflows.
```

Comment:
Also, I’ve added a general change to pull out our workflows into reusable chunks. The GitHub UI sort of rewards you with a better UX for doing this (e.g., better graphical representation of what’s happening).